### PR TITLE
Java_Program_to_Find_Factorials_Recursion

### PR DESCRIPTION
**Pull request by Buddhimah**
Please review my code

### CODE REVIEW

File: java/recursion/Factorial.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+/**
+ * Created by buddhimah on 10/16/2017.
+ */
+public class Factorial {
+
+    public static  int fac(int num){
+
+        int r;
+
+        if(num==1){
```
Comment (utkarsh-shekhar) [inline java/recursion/Factorial.java]: Handle negative cases by raising an error or by printing that factorial for negative numbers is undefined.

File: java/recursion/NCR.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+/**
+ * Created by buddhimah on 10/16/2017.
+ */
+public class NCR {
+    public static  int fac(int num){
+
+        int r;
+
+        if(num==1){
```
Comment (utkarsh-shekhar) [inline java/recursion/NCR.java]: Handle negative cases.

Comment (Buddhimah) [reply java/recursion/NCR.java]: Thank you I will handle it

Comment (Buddhimah) [reply java/recursion/NCR.java]: Its Done :)

Comment (Buddhimah) [reply java/recursion/NCR.java]: @utkarsh-shekhar The change request you asked for completed

**Pull request merged**